cmd/oras/push.go

```diff
@@ -99,6 +104,9 @@ Example - Push file "hi.txt" with multiple tags and concurrency level tuned:
 			return option.Parse(&opts)
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.ManifestMediaType == ocispec.MediaTypeArtifactManifest && opts.manifestConfigRef != "" {
+				return errors.New("cannot pack an OCI artifact with manifest config at the same time")
+			}
```

Comment:
It reminds me that there will be a feature request eventually to support `--config` for `oras attach`.

Reply:
Why? The content of config doesn't matter and the media type of config can be deducted from artifact type.

Reply:
If they could, they will ask.